contracts/src/AccountRegistry.sol

```diff
@@ -34,7 +25,7 @@ contract AccountRegistry is Initializable, EIP712Upgradeable, Ownable2StepUpgrad
     //                        Members                         //
     ////////////////////////////////////////////////////////////
 
-    // accountIndex -> [96 bits bitmap of pubkeyIds][160 bits recoveryAddress]
+    // accountIndex -> [32 bits pubkeyId bitmap][64 bits unused][160 bits recoveryAddress]
```

Comment:
Is there maybe a confusion here of pubkeyIds and the bitmap of pubkeys or why reducing this? This also has the effect that there can only be 32 pubkeys registered.